clang/lib/CIR/Dialect/Transforms/LoweringPrepare.cpp

```diff
@@ -354,6 +355,18 @@ void LoweringPreparePass::lowerVAArgOp(VAArgOp op) {
   return;
 }
 
+void LoweringPreparePass::lowerDeleteArrayOp(DeleteArrayOp op) {
+  CIRBaseBuilderTy builder(getContext());
+  builder.setInsertionPoint(op);
+  cir::CIRDataLayout datalayout(theModule);
```

Comment:
I see we have many of these `datalayout` in different methods in LoweringPrepare, can we make this a class member and initialize it once?

Reply:
Done